tests/installation/await_install.pm

```diff
@@ -125,6 +125,13 @@ sub run {
     }
     # on s390 we might need to install additional packages depending on the installation method
     if (check_var('ARCH', 's390x')) {
+      # For poo#93949, we need password possibility
+      if (!is_sle && !is_leap) {
+        select_console 'install-shell';
+        assert_script_run("mkdir -p /mnt/etc/ssh/sshd_config.d");
+        assert_script_run("(echo 'PermitRootLogin yes'; echo 'PasswordAuthentication yes'; echo 'ChallengeResponseAuthentication yes') > /mnt/etc/ssh/sshd_config.d/allow-root-with-password.conf");
```

Comment:
Should be enough:

```suggestion
        assert_script_run('echo PermitRootLogin yes > /mnt/etc/ssh/sshd_config.d/allow-root-with-password.conf');
```

Reply:
Really? Then the default path (created by SUSE QA) should have worked, too. That was not the case 1 month ago. I test it again....